tests/Fixer/ControlStructure/SimplifiedIfReturnFixerTest.php

```diff
@@ -0,0 +1,250 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\ControlStructure;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\ControlStructure\SimplifiedIfReturnFixer
+ */
+final class SimplifiedIfReturnFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return [
+            'simple' => [
+                '<?php return (bool) ($foo)      ;',
+                '<?php if ($foo) { return true; } return false;',
+            ],
+            'simple-negative' => [
+                '<?php return ! ($foo)      ;',
+                '<?php if ($foo) { return false; } return true;',
+            ],
+            'simple-braceless' => [
+                '<?php return (bool) ($foo)    ;',
+                '<?php if ($foo) return true; return false;',
+            ],
+            'simple-braceless-negative' => [
+                '<?php return ! ($foo)    ;',
+                '<?php if ($foo) return false; return true;',
+            ],
+            'bug1' => [
```

Comment:
What's "bug1"?

Reply:
I don't remember: I do remember this test case exposed a bug I found in an earlier implementation, and yesterday I added array keys to the data provide as a refactoring practise

Reply:
Can we change the name to something more useful?

Reply:
Of course, but I'm not able to think any meaningful phrase, as I don't know what the test case is testing :\

Reply:
Maybe `consecutive_ifs`? Looks like those are the only test cases covering this scenario.

Reply:
Done, thank you for the suggestion :+1: